DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/ViewTreeRecorder.swift

```diff
@@ -52,12 +62,7 @@ internal struct ViewTreeRecorder {
         }
     }
 
-    private func nodeSemantics(for view: UIView, in context: ViewTreeRecordingContext) -> NodeSemantics {
-        let attributes = ViewAttributes(
-            frameInRootView: view.convert(view.bounds, to: context.coordinateSpace),
-            view: view
-        )
-
+    private func nodeSemantics(for view: UIView, with attributes: ViewAttributes, in context: ViewTreeRecordingContext) -> NodeSemantics {
```

Comment:
You need to rebase because of my feature branch recent merge. This method also takes a `PrivacyOverrides` argument to create the `ViewAttributes`. But that shouldn't be a problem, you can still create the `ViewAttributes` in `recordRecursively`.